app/src/main/res/layout/audio_fragment.xml

```diff
@@ -72,10 +71,8 @@
       android:paddingTop="16dp"
       android:paddingEnd="16dp"
       android:paddingBottom="16dp"
-      android:text="@{viewModel.currentLanguageCode}"
-      android:textAllCaps="true"
-      android:textColor="@color/white"
-      android:textSize="14sp"
+      android:contentDescription="@{viewModel.currentLanguageCode}"
```

Comment:
To maintain the code format, we always use `Reformat code` option from the menu.

After reformatting this `ImageView` will be like this:

```
<ImageView
      android:id="@+id/tvAudioLanguage"
      android:layout_width="wrap_content"
      android:layout_height="wrap_content"
      android:contentDescription="@{viewModel.currentLanguageCode}"
      android:gravity="center"
      android:minWidth="48dp"
      android:minHeight="48dp"
      android:onClick="@{(v) -> audioFragment.languageSelectionClicked()}"
      android:paddingStart="8dp"
      android:paddingTop="16dp"
      android:paddingEnd="16dp"
      android:paddingBottom="16dp"
      android:src="@drawable/ic_audio_lang_24px"
      app:layout_constraintBottom_toBottomOf="parent"
      app:layout_constraintEnd_toEndOf="parent"
      app:layout_constraintTop_toTopOf="parent" />
```

Reply:
done